### ISSUE TITLE
larger epoch than other methods

### ISSUE THREAD

**Issue by dreamcubeblock** [zyh16143998882/LCM#2]
You are using a much larger epoch number than other methods (Point-MAE, ACT, ReCon, PCP-MAE, etc.). Could you tell me why? 

I have reproduced your method on 300 epochs, and it seems to perform not that well.

**Comment by zyh161439** [comment]
We propose a novel backbone that differs from the original Transformer architecture. Due to the nature of this backbone, it requires more epochs to converge under the current learning rate. For a fair comparison, we also report the performance of Transformer-based architectures trained with the same number of epochs (1000) in our paper. These Transformer results are obtained under our downstream settings and are highlighted in orange in Table 1.
